CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2022-11-17
+| What's new | 
+| :--------- |
+| We have improved the performance of the Fields page to better support graphs with a large number of fields (> 1000).
```

Comment:
I'd say if we include this change that we should be super explicit about fixing _part_ of the problem but not all of the problem, and that we're working on solutions to better handle the other issues.  Alternatively, wait until we've finished the improvements to handling the stats loading / errors UX and announce them together.